package.json

```diff
@@ -21,18 +21,17 @@
     "dist:zip": "zip -r dist.zip . --exclude @.funcignore --exclude .funcignore",
     "predeploy": "npm-run-all generate build dist:modules",
     "generate:definitions": "shx rm -rf ./generated/definitions && shx mkdir -p ./generated/definitions && gen-api-models --api-spec ./openapi/index.yaml --out-dir ./generated/definitions",
-    "generate": "npm-run-all generate:* generate:ts-from-yaml:*",
-    "generate:ts-from-yaml:SvcConsultazione": "shx mkdir -p generated/SvcConsultazione && rimraf generated/SvcConsultazione/* && gen-api-models --strict 0 --api-spec generated/yaml-file/SvcConsultazione.yaml --out-dir generated/SvcConsultazione",
-    "generate:yaml-from-wsdl": "shx mkdir -p generated/yaml-file && rimraf generated/yaml-file/* && node generator/yamlGenerator.js"
+    "generate": "npm-run-all generate:*"
   },
   "devDependencies": {
     "@azure/functions": "^1.0.3",
     "@types/bluebird": "^3.5.32",
     "@types/documentdb": "^1.10.5",
     "@types/express": "^4.16.0",
     "@types/jest": "^24.0.15",
-    "@types/soap": "^0.21.0",
+    "@types/node": "^14.0.5",
```

Comment:
Should we use the same major/minor of the actual node runtime?

Reply:
fixed in 0c41302